src/components/Restorans/Nav/Nav.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useNavigate } from 'react-router';
+import RestoranIcon from '../../../images/Vector.png';
+import Notepad from '../../../images/notepad.png';
+import Star from '../../../images/star.png';
+import Picture from '../../../images/picture.png';
+
+const Nav = () => {
+  const navigate = useNavigate();
+  const navigateToMeni = () => {
+    navigate('/meni');
+  };
+  const navigateToRestoran = () => {
+    navigate('/topfood');
+  };
+  const navigateToGalery = () => {
+    navigate('galery');
```

Comment:
You are missing sign `/` , cause when you click on button gallery you won't navigate to gallery page. In fact you will be redirected to `localhost:3000/{previous-page/gallery}`, which is missing. Also you made typo here you should change from `galery` to `gallery`

Reply:
Add / to navigate('/galery');